### PR TITLE
Mansion Scenario

### PR DESCRIPTION
**Pull request by Termineitor244**
#### Summary
Content "Adds a new scenario for starting in a mansion"

#### Purpose of change

Someone on reddit reminded me of an old mod that added a mansion scenario, I searched it and it was removed because it became obsolete... But you still can't start the game in a mansion! With the mansions basecamps and the good loot and roleplaying related to mansions, I think there is enough reason to add one.

#### Describe the solution

Adds a new mansion scenario, inspired by the one in Classes and Scenarios Mod, but with a different description and more professions (And it works now). 

#### Describe alternatives you've considered

Adding more options for where to start in the mansion, like the basement or the upper floor, or even the Panic Room! But for now this is enough.
I also contemplated to simply add mansions as a possible location for the "Big Building" scenario, but scenarios are too cool for that.

#### Testing

I spawned several times in a mansion without problems, looked around, killed some zombies, and died at the hands of the old staff of the mansion!

#### Additional context

imagen
imagen
imagen

### CODE REVIEW

**Comment by ZhilkinSerg** [comment]
New scenarios should really do something more interesting than simply allowing player character to start at specific location, e.g.:

- add custom start mission (https://github.com/CleverRaven/Cataclysm-DDA/pull/39703);
- use custom surround group (https://github.com/CleverRaven/Cataclysm-DDA/pull/47990);
- use power of eocs (https://github.com/CleverRaven/Cataclysm-DDA/pull/50934).

**Comment by Termineitor244** [comment]
> 
> 
> New scenarios should really do something more interesting than simply allowing player character to start at specific location, e.g.:
> 
>     * add custom start mission ([ Add new starting scenario The Last Delivery  #39703](https://github.com/CleverRaven/Cataclysm-DDA/pull/39703));
> 
>     * use custom surround group ([Allow custom surround monster groups in scenarios #47990](https://github.com/CleverRaven/Cataclysm-DDA/pull/47990));
> 
>     * use power of eocs ([Scenario specific effects on condition #50934](https://github.com/CleverRaven/Cataclysm-DDA/pull/50934)).

I could do a custom surround group for the scenario... Maybe some eocs, missions... I don't have ideas for one, but lets see what happens.

I will put the PR in draft while I do these changes then.

**Comment by Maleclypse** [comment]
Might I suggest the movie Ready or Not (2019 film) for some possible inspiration?

**Comment by Termineitor244** [comment]
> Ready or Not

I had something completely different (and more simple) in mind, but hahaha, after reading the wikipedia article of the movie, I think I want some new mansion ferals to start surrounding you in this scenario, and a new mission to survive until next day, I will give it a try and see what comes out of it, thanks for the idea!

**Comment by Termineitor244** [comment]
I added new ferals, the only really dangerous are the one with the crossbow and the real rapier, but they should give even more uniqueness to mansions, I will add them as a rare spawn to the monstergroups of mansions too, and the fancy ones will have a greater possibility in the spawn for the panic room (Makes sense).

I tried adding a mission to survive for the day, but it doesn't seems to be working... It doesn't finish after 1, 2 or 3 days, I'm not sure why.

I want to write a different description for the scenario "you were in a mansion, playing hide and seek with the staff/owners, and suddenly they are trying to kill you whenever they found you" something like that, but I haven't worded it right yet.

I would like to use eocs to spawn a weapon with the player when they start the scenario (You managed to steal a shotgun/medieval weapon when you were hiding), suggestions for the weapon in question?

File: data/json/monsters/feral_humans.json [hunk 1]
```diff
@@ -215,5 +215,101 @@
...
+    "melee_skill": 3,
+    "melee_cut": 1,
+    "death_drops": "feral_fancy_death_drops_rapier_fake"
+  },
+  {
+    "id": "mon_feral_fancy_crossbow",
+    "type": "MONSTER",
+    "copy-from": "mon_feral_maid_broom",
+    "name": { "str": "well dressed feral" },
+    "description": "Wearing fancy clothes and with a crossbow in hand, this maniac was once a socialite.  Its hands seem to wander trough the crossbow as if touching a loved one.",
```
Comment (Maleclypse) [inline data/json/monsters/feral_humans.json:296]:
```suggestion
    "description": "Wearing fancy clothes and with a crossbow in hand, this maniac was once a socialite.  Its hands seem to wander the crossbow as if touching a loved one.",
```

**Comment by Termineitor244** [comment]
Agh... I will close this and open an new PR with the changes

**Pull request closed without merging**